admin/app/transforms/date-only.js

```diff
@@ -0,0 +1,11 @@
+import DS from 'ember-data';
+import moment from 'moment';
+
+export default DS.Transform.extend({
+  serialize: function(date) {
+    return moment(date).format('YYYY-MM-DD');
+  },
+  deserialize: function(date) {
+    return moment(date).format('YYYY-MM-DD');
+  }
+});
```

Comment:
Le titre du commit est "Handle DATE type as 'YYYY-MM-DD' in the API" et pourtant ce fichier est ajouté dans Pix Admin ? 🤔Je pense qu'il n'est pas dans le bon commit 😅

Reply:
Oui merci, rebasage foiré :smile_cat: